libs/components/behavioral/src/assets/landing-page-content.yml

```diff
@@ -0,0 +1,54 @@
+LandingPageIntroTitle:
+  type: markdown
+  markdown: 'Welcome to the Functional Tissue Unit Explorer'
+LandingPageIntroDescription:
+  type: markdown
+  markdown: "A functional tissue unit (FTU) is the smallest tissue organization that performs a unique physiologic function and is replicated multiple times in a whole organ. Explore medical illustrations of FTUs to view cell type populations by gene, protein, and lipid biomarkers. The FTU Explorer features experimental datasets from <a href='' target='_blank'>HuBMAP</a>, <a href='' target='_blank'>CellxGene</a>, <a href='' target='_blank'>NeMO</a>, and <a href='' target='_blank'>GTEx</a>."
+LandingPageIntroPartners:
+  type: markdown
+  markdown: "A special thanks to our partners: The FTU Explorer was designed in close collaboration with <a href='' target='_blank'>Kidney Precision Medicine Project</a> and <a href=''>European Bioinformatics Institute.</a>"
+LandingPageIntroMoreText:
+  type: markdown
+  markdown: 'Explore FTUs'
+LandingPageIntroImg:
+  type: url
+  url: 'assets/welcome.svg'
+MetricsLogo:
+  type: url
+  url: '/assets/logo.svg'
```

Comment:
Remove `/` before assets or it will not work in deploy. Multiple urls needs to be updated